pkg/integration/utils.go

```diff
@@ -61,12 +78,38 @@ func IsKilled(err error) bool {
 
 // RunCommandWithOutput runs the specified command and returns the combined output (stdout/stderr)
 // with the exitCode different from 0 and the error if something bad happened
-func RunCommandWithOutput(cmd *exec.Cmd) (output string, exitCode int, err error) {
-	exitCode = 0
-	out, err := cmd.CombinedOutput()
-	exitCode = ProcessExitCode(err)
-	output = string(out)
-	return
+func RunCommandWithOutput(cmd *exec.Cmd) (string, int, error) {
+	// TODO Windows CI, TP5 timeframe, once windowsTP4 context does not
+	// exist in Jenkins, remove this retry hack logic.
+	var (
+		output   string
+		exitCode int
+		err      error
+		out      []byte
+	)
+
+	maxAttempts := 1
+	if windowsRetryHack {
+		maxAttempts = 10
+	}
+
+	i := 0
+	savedCmd := *cmd
+	for i < maxAttempts {
+		i++
+		exitCode = 0
+		out, err = cmd.CombinedOutput()
+		exitCode = ProcessExitCode(err)
+		output = string(out)
+		if windowsRetryHack {
+			if !strings.Contains(output, `HCSShim::CreateComputeSystem - Win32 API call returned error r1=2147746291 err=Invalid class string`) {
+				return output, exitCode, err
+			}
+			cmd = &savedCmd // As cmd cannot be reused
```

Comment:
I see that it's a copy in the first run, but you only make a copy once. Or is it safe to use it multiple time if it's a copy.